extension/e2e-tests/addAsset.test.ts

```diff
@@ -6,14 +6,15 @@ test("Adding unverified Soroban token", async ({ page, extensionId }) => {
   test.slow();
   await loginToTestAccount({ page, extensionId });
 
+  await page.getByTestId("account-options-dropdown").click();
   await page.getByText("Manage Assets").click({ force: true });
   await page.getByPlaceholder("Enter password").fill(PASSWORD);
   await page.getByText("Log In").click({ force: true });
+  await expect(page.getByText("Your assets")).toBeVisible();
```

Comment:
This fixes a bug with snapshot: https://github.com/stellar/freighter/pull/1580/files#diff-93b044b86208b123f27589535905ad725da70f6b467ee0559c988a9f866e71e1

It was mistakenly snapshotting the password confirm screen instead of the manage assets screen because it wasn't waiting for the view to load